samples/encode-decode-worker/js/main.js

```diff
@@ -255,7 +255,7 @@ document.addEventListener('DOMContentLoaded', async function(event) {
           config.pt = 1;
           break;
         case "H265":
-          config.codec = "hvc1.1.6.L123.00" // Main profile
+          config.codec = "hvc1.1.6.L123.00"  // Main profile, level 4.1, main Tier
           config.hevc = { format: "annexb" };
           config.pt = 2;
           addToEventLog('HEVC Encoding not supported', 'fatal');
```

Comment:
we already get chromium on Windows supporting HEVC encoding behind a flag. Could you consider breaking here instead of returning directly?

Reply:
Fixed.